front_end/src/app/(main)/questions/[id]/components/forecast_maker/forecast_maker_question/forecast_maker_binary.tsx

```diff
@@ -85,6 +89,29 @@ const ForecastMakerBinary: FC<Props> = ({
     }
   };
   const [submit, isPending] = useServerAction(handlePredictSubmit);
+
+  const [withdrawError, setWithdrawError] = useState<ErrorResponse>();
+  const handlePredictWithdraw = async () => {
+    setWithdrawError(undefined);
+
+    if (!prevForecastValue) return;
+
+    const response = await withdrawForecasts(post.id, [
+      {
+        question: question.id,
+        // withdrawAt: new Date().toISOString(), // TODO: implement
```

Comment:
The plan is to address this TODO in a separate PR?

Reply:
Yes. It's a future-proofing feature (pre-registering prediction withdrawal).
I suppose I could just remove the comment.